### DISCUSSION TITLE
Missing files Comsol example

### DISCUSSION THREAD

**Discussion by robin2715** [pybamm-team/PyBaMM#2165] [Q&A] [answered]
Hi

I am interested to see a comparative result between Comsol and Pybamm. Therefore, I tried to run the example located in [compare_comsol_DFN.py](https://github.com/pybamm-team/PyBaMM/blob/main/examples/scripts/compare_comsol/compare_comsol_DFN.py). 

However, when I tried to run the example, I get a FileNotFoundError pointing at 
'C:\\ProgramData\\Miniconda3\\lib\\site-packages\\pybamm\\input/comsol_results/comsol_1C.pickle'. 
The folder 'comsol_result' is indeed not there.  Any hint of how to proceed?

Thanks in advance for any help

Róbinson Medina

Some additional data:
- My Pybamm installation works fine with other examples. 
- I am using windows with an anaconda environment.

**Comment by brosaplanella** [comment, marked as answer]
If they are not there you can [download the files from Github](https://github.com/pybamm-team/PyBaMM/tree/develop/pybamm/input/comsol_results) and then amend the path in your code so it finds the file.

**Reply by robin2715** [reply]
Thanks!
